airbyte-integrations/connectors/source-asana/source_asana/spec.json

```diff
@@ -65,12 +65,48 @@
       }
     }
   },
-  "authSpecification": {
-    "auth_type": "oauth2.0",
-    "oauth2Specification": {
-      "rootObject": ["credentials", 1],
-      "oauthFlowInitParameters": [["client_id"], ["client_secret"]],
-      "oauthFlowOutputParameters": [["refresh_token"]]
+  "advanced_auth": {
```

Comment:
you need to add predicate key and predicate value

Reply:
@arsenlosenko did you check it manually?

Reply:
not yet, checking it right now, I'll request review again once all things are checked and my changes are updated